File: app-serverless/src/main/java/com/xsolla/android/storesdkexample/util/ViewUtils.java
package com.xsolla.android.storesdkexample.util;

import android.view.View;

public class ViewUtils {

    private static final long DELAY_TIME = 1000L;

    public static void disable(View view) {
        view.setEnabled(false);
    }

    public static void enable(View view) {
        view.setEnabled(true);
    }
}

File: app/src/main/java/com/xsolla/android/storesdkexample/util/ViewUtils.kt
package com.xsolla.android.storesdkexample.util

import android.view.View

object ViewUtils {
    private const val DELAY_TIME = 1000L

    fun disable(view: View) {
        view.isEnabled = false
    }

    fun enable(view: View) {
        view.isEnabled = true
    }
}